src/plugin.yml

```diff
@@ -24,6 +24,7 @@ permissions:
       bending.ability.AvatarState: true
       bending.water.bloodbending.anytime: true
       bending.water.bloodbending: true
+      bending.fire.blue: true
```

Comment:
sir please follow proper naming convention we have mostly established, `bending.fire.bluefire`

Reply:
hehehe yeah.